src/main/java/mekanism/common/FuelHandler.java

```diff
@@ -56,7 +58,7 @@ public FuelGas(int duration, double energyDensity)
 		public FuelGas(IFuel bcFuel)
 		{
 			burnTicks = bcFuel.getTotalBurningTime() / Fluid.BUCKET_VOLUME;
-			energyPerTick = bcFuel.getPowerPerCycle() * general.FROM_RF;
+			energyPerTick = bcFuel.getPowerPerCycle() * MjAPI.MJ;
```

Comment:
Is there a way to get the BCmj -> RF rate from the API? I believe Mek has separate conversion config values, but we could at least use that to generate a default value.